mlcustomize/inject_virtio_win.ml

```diff
@@ -492,37 +492,37 @@ and virtio_iso_path_matches_guest_os t path =
     and any_variant os_variant = true
     and any_osinfo osinfo = true in
     let p_os_major, p_os_minor, match_os_variant, match_osinfo =
-      if pathelem "xp" || pathelem "winxp" then
+      if pathelem "xp" then
         (5, 1, any_variant, any_osinfo)
-      else if pathelem "2k3" || pathelem "win2003" then
+      else if pathelem "2k3" then
         (5, 2, any_variant, any_osinfo)
       else if pathelem "vista" then
         (6, 0, is_client, any_osinfo)
-      else if pathelem "2k8" || pathelem "win2008" then
+      else if pathelem "2k8" then
         (6, 0, not_client, any_osinfo)
-      else if pathelem "w7" || pathelem "win7" then
+      else if pathelem "w7" then
```

Comment:
I don't think this is right.  My `virtio-win-1.9.41-1.el9.noarch` has files named:
`/usr/share/virtio-win/drivers/amd64/Win7/*`
where a path element would match `win7` (the path element is lowercased when we compare it).

I can also see `win8` and others appearing.

Reply:
I explain that in the commit message.

Reply:
FWIW I also sent a virtio-win packaging PR that deletes `drivers/$ARCH` since it's been redundant since `by-os/` was introduced in 2020. https://github.com/virtio-win/virtio-win-pkg-scripts/pull/102

Reply:
OK I see, thanks.